### DISCUSSION TITLE
Banana Cake Pop RUN  Button diasappears

### DISCUSSION THREAD

**Discussion by AivaloCap** [ChilliCream/hotchocolate#5956] [Q&A]
After a while, the button RUN in each query has disappeared. Then, right now, I need to open a new tab if I want to run each query individually. 

How can I see the button again? 

![image](https://user-images.githubusercontent.com/84666783/224677422-20dc93f6-4804-413f-aae6-dd56cb184a22.png)

**Comment by PascalSenn** [comment]
This is probably because your document is invalid.
Can you post your document here?

**Reply by AivaloCap** [reply]
I mean, It's running well if I run with the Run button for all the queries.
It's not just only with one document, and before I could see it using the same documents. 

![image](https://user-images.githubusercontent.com/84666783/224760740-99f83b6d-4586-4fd4-85ce-4adcc51fe9fe.png)

**Comment by artola** [comment]
@AivaloCap In general, if the document (all the text, operations) is valid (well written and we have the schema to validate it), the _Codelens_ (**Run**) should be shown asap. Every time that there is a change, we re-evaluate them.
If by some reason they disappear, as you mention above, try just adding some spaces to force the process (or click Schema Reference/Definition tabs back and forth, or any other open tab).
By the way, which version of BCP are you running? (I was not able to reproduce it on v4.0.1 and v5.0.0-insider.5).

**Reply by AivaloCap** [reply]
![image](https://user-images.githubusercontent.com/84666783/224991455-aa28e0d2-1484-43f4-a088-b75fc54057b3.png)
It's v4.0.0.
Don't work your tricks. Another colleague is having the same issue with this code on his laptop.  
BTW Thank you for your help.

**Reply by AivaloCap** [reply]
I've updated to version 4.0.1 and I'm having the same issue. 
Actually, the problem is happening with Google Chrome, cause I've tried to swap to Firefox and it's working perfectly.
